app/src/main/res/layout/cover_fragment.xml

```diff
@@ -1,19 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<android.support.percent.PercentRelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/cover_fragment_root"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical" >
+    android:background="@color/actionbar_gray">
 
     <ImageView
         android:id="@+id/imgvCover"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_centerInParent="true"
         android:contentDescription="@string/cover_label"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_gravity="center"
-        android:adjustViewBounds="true"
         android:scaleType="fitCenter"
+        app:layout_aspectRatio="100%"
+        app:layout_widthPercent="82%"
         tools:src="@android:drawable/sym_def_app_icon" />
 
-</RelativeLayout>
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center"
+        android:orientation="vertical"
+        android:layout_below="@id/imgvCover">
+
+        <TextView
+            android:id="@+id/txtvEpisodeTitle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Episode" />
+
+        <TextView
+            android:id="@+id/txtvPodcastTitle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Podcast" />
```

Comment:
Text looks really ugly when it wraps here:

![device-2016-01-09-112811](https://cloud.githubusercontent.com/assets/13721872/12217395/669cc17e-b6c4-11e5-980c-6498bc5b790f.png)

Add these lines to the episode and podcast title TextViews to fix it:

    android:layout_marginLeft="8dp"
    android:layout_marginRight="8dp"
    android:layout_marginStart="8dp"
    android:layout_marginEnd="8dp"
    android:gravity="center"